Felienne/hedy
# [BUG FIX] Visual Studio download link works again

**Description**
We update the download link for Visual Studio so it works again.

**Fixes**
This PR fixes #2854.

**How to test**
Click the link, verify that you are being redirected to the Visual Studio download page.
